### PR TITLE
feat: integration of scss and change layout styles

### PR DESCRIPTION
**Pull request by Kovalenko77**

### CODE REVIEW

File: scss/style.scss
Comment (romanalesenkov1) [inline scss/style.scss:1]: why do you need this empty file?

File: src/index.scss [hunk 1]
```diff
@@ -89,3 +99,12 @@ td {
 .download-button {
   background-color: #307d39;
 }
+
+input {
+  @extend .input__style;
```
Comment (romanalesenkov1) [inline src/index.scss]: why do you need it?

File: scss/style.scss
Comment (Kovalenko77) [reply scss/style.scss:1]: No need, removed. Thank you.

File: src/index.scss (hunk 1 above)
Comment (Kovalenko77) [reply src/index.scss]: No need, removed. Thank you.

**Pull request merged**